README.rst

```diff
@@ -88,7 +130,7 @@ Testing
 After installation, you can launch the test suite from inside the
 source directory   ::
 
-    pytest
+    pytest gph
```

Comment:
@MonkeyBreaker @matteocao does this change sound right to you?

Reply:
https://docs.pytest.org/en/latest/how-to/usage.html#calling-pytest-through-python-m-pytest  what do you think?  It's not clear to me

Reply:
https://stackoverflow.com/a/66090281

It does not seem to impact us. It was more to standardized in the documentation when we call python modules.

Reply:
Ok, maybe we leave it then :)

Reply:
About adding ``gph``, I just felt it would make it clearer that this is a command and not a typo.

Reply:
Then all good to me :+1: 
I just like to type less, that is way I personally prefer `pytest`